nntrainer/tensor/var_grad.cpp

```diff
@@ -24,20 +24,21 @@ Var_Grad::Var_Grad(const TensorDim &dim, bool train, const std::string &name) :
   grad = std::make_shared<Tensor>();
 }
 
-void Var_Grad::initialize(const Tensor &weights_preallocated,
-                          const Tensor &grad_preallocated, bool gtrain) {
-  if (!weights_preallocated.uninitialized()) {
-    var = std::make_shared<Tensor>(weights_preallocated);
+void Var_Grad::initializeWeight(const Tensor &preallocated, bool gtrain) {
+  if (!preallocated.uninitialized()) {
```

Comment:
we do not need this gtrain here.

Reply:
`gtrain` is kept here for completeness and to maintain the same interface as `initializeGradient`

Reply:
Is there any reason to maintain same interface with Gradient ?

Reply:
No advantage, other than ease of reading.

Reply:
Well, I don't think it makes easy. It might be more like miss leading that it is going to use somewhere. It would be better to delete.

Reply:
done.